opencti-platform/opencti-graphql/src/database/middleware-loader.ts

```diff
@@ -456,8 +456,8 @@ export const pageEntitiesConnection = async <T extends BasicStoreEntity>(context
   // maxSize MUST be aligned with first in this method.
   // As using elConnection is repaginate, removing maxSize will lead to major api breaking
   const paginateArgs = { ...buildEntityFilters(entityTypes, args), first, maxSize: first };
-  const { elements, totalCount, totalFilteredCount } = await elConnection(context, user, computedIndices, paginateArgs);
-  return buildPaginationFromEdges(args.first, args.after, elements, totalCount, totalFilteredCount);
+  const { elements, totalCount } = await elConnection(context, user, computedIndices, paginateArgs);
```

Comment:
in fact elConnection is not returning a 'connection' format. shouldn't we align elConnection ouput with other '*Connection' so here we would only do a return elConnection(...) ?